docs/modules/mobile/prepare-project.md

```diff
@@ -27,9 +27,9 @@ To connect to a cloud deployment using your mobile device, you need to use the [
 1. In the **Dev Auth Token Settings** specify the lifetime of the token and the path to the [Resources folder](https://unity3d.com/learn/tutorials/topics/best-practices/resources-folder) that you would like to store the generated token in.
 
     > Your token expires after the amount of days that you specified in the configuration window. Regenerate the token whenever that happens.
-
+1. (for iOS) Ensure that **Save token to file** is checked.
 1. Select **Save** and close the window.
-1. Select **SpatialOS** > **Generate Dev Authentication Token**. This generates a `DevAuthToken.txt` asset in the folder you specified in the configuration window. This token is used to authenticate when connecting to a cloud deployment.
+1. Select **SpatialOS** > **Generate Dev Authentication Token**. This stores the dev auth token inside the Unity Player Preferences. If **Save token to file** inside the configuration window is checked, it will also generate a `DevAuthToken.txt` asset in the folder you specified in the configuration window. This token is used to authenticate when connecting to a cloud deployment.
```

Comment:
actually it's **SpOS** > **Dev Authentication Token** > **Generate Token** now 😬

Reply:
should probably update this if there are other places we tell people to generate a token

Reply:
seems to be the only place